crates/core/src/db/queries/metadatas.rs

```diff
@@ -426,6 +445,230 @@ pub fn list(
         .context("Failed to load nft(s)")
 }
 
+/// Input parameters for the [`collection_nfts`] query.
+#[derive(Debug)]
+pub struct CollectionNftOptions {
+    /// Collection address
+    pub collection: String,
+    /// Auction house of the collection
+    pub auction_house: Option<String>,
+    /// Filter by collection attributes
+    pub attributes: Option<Vec<AttributeFilter>>,
+    /// Marketplace program in which the collection is listed
+    pub marketplace_program: Option<String>,
+    /// Sort by Price or Listed at
+    pub sort_by: Option<Sort>,
+    /// Order the resulting rows by 'Asc' or 'Desc'
+    pub order: Option<Order>,
+    /// Limit the number of returned rows
+    pub limit: u64,
+    /// Skip the first `n` resulting rows
+    pub offset: u64,
+}
+
+/// Handles queries for a Collection Nfts
+///
+/// # Errors
+/// returns an error when the underlying queries throw an error
+#[allow(clippy::too_many_lines)]
+pub fn collection_nfts(conn: &Connection, options: CollectionNftOptions) -> Result<Vec<Nft>> {
+    let CollectionNftOptions {
+        collection,
+        auction_house,
+        attributes,
+        marketplace_program,
+        sort_by,
+        order,
+        limit,
+        offset,
+    } = options;
+    let current_time = Utc::now().naive_utc();
+
+    let sort_unwrap = match sort_by.unwrap() {
+        Sort::Price => Listings::Price,
+        Sort::ListedAt => Listings::CreatedAt,
+    };
+
+    let order_unwrap = order.unwrap();
```

Comment:
```suggestion
    let order_by = order.unwrap_or(Order::Desc)
```
Set desc as the default.